File: ifthenelse.erl
% ifthenelse.erl
-module(ifthenelse).
-export([main/0]).

main() ->
    X = 5,
    if X > 6 ->
        io:format("x is greater than 6~n");
    true ->
        io:format("x is less than or equal to 6~n")
    end.

File: ifthenelse.f90
! ifthenelse.f90
program ifthenelse
    implicit none
    integer :: x
    x = 5

    if (x > 6) then
        print *, "x is greater than 6"
    else
        print *, "x is less than or equal to 6"
    end if
end program ifthenelse
